src/verinfast/dependencies/walkers/npm.py

```diff
@@ -19,12 +19,16 @@ def initialize(self, root_path: str = "./"):
         for p in self.install_points:
             target_dir = Path(p).parent
             os.chdir(target_dir)
-            subprocess.run(
-                "npm install --production",
-                capture_output=True,
-                shell=True
-            )
-            self.walk('node_modules')
+            try:
+                subprocess.run(
+                    "npm install --production --silent --yes",
+                    capture_output=True,
+                    shell=True
+                )
+            except Exception as error:
+                self.log(f'Error with npm install: {error}')
```

Comment:
We should start surfacing the scan logs in the UI

Reply:
Maybe a --verbose flag? Some logs are surfaced with self.log(display=true)

Reply:
sorry, I meant atd UI